From f422449b58548a41e98fc97b259a283718e527db Mon Sep 17 00:00:00 2001
From: Cheng-Min Ao <tony_ao@wiwynn.com>
Date: Mon, 7 Jan 2019 14:29:32 +0800
Subject: [PATCH] hwmon: (tmp421) Correct the misspelling of the tmp442 compatible attribute in OF device ID table
Git-commit: f422449b58548a41e98fc97b259a283718e527db
Patch-mainline: v5.0-rc3
References: bsc#1051510

Correct a typo in OF device ID table
The last one should be 'ti,tmp442'

Signed-off-by: Cheng-Min Ao <tony_ao@wiwynn.com>
Signed-off-by: Yu-Hsiang Chen <matt_chen@wiwynn.com>
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/hwmon/tmp421.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hwmon/tmp421.c b/drivers/hwmon/tmp421.c
index 8844c9565d2a..7053be59ad2e 100644
--- a/drivers/hwmon/tmp421.c
+++ b/drivers/hwmon/tmp421.c
@@ -88,7 +88,7 @@ static const struct of_device_id tmp421_of_match[] = {
 		.data = (void *)2
 	},
 	{
-		.compatible = "ti,tmp422",
+		.compatible = "ti,tmp442",
 		.data = (void *)3
 	},
 	{ },
-- 
2.16.4

